pkg/extensions/extension_minimal.go

```diff
@@ -0,0 +1,26 @@
+// +build minimal
+
+package extensions
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/anuvu/zot/pkg/log"
+	"github.com/anuvu/zot/pkg/storage"
+)
+
+// ExtensionServer ...
+type ExtensionServer struct {
+	GraphqlServer http.Handler
```

Comment:
Is this needed at all?

Reply:
Yes, see routes.go we need graphql server when we enable cve scan and when we want to generate new minimal binary 
ExtensionServer need to be passed.